lib/cisco_node_utils/bgp_neighbor.rb

```diff
@@ -431,7 +432,7 @@ def default_transport_passive_mode
 
     def transport_passive_only=(val)
       # TODO: add "deprecated" warning?
-      self.transport_passive_mode = (val ? :passive_only : :both)
+      self.transport_passive_mode = (val ? :passive_only : :none)
```

Comment:
The transport_passive_mode setter also has a :nexus check.  Maybe you are looking at the transpor_passive_only setter?